packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesFileDownloader.swift

```diff
@@ -0,0 +1,823 @@
+//  Copyright © 2019 650 Industries. All rights reserved.
+
+// swiftlint:disable closure_body_length
+// swiftlint:disable file_length
+// swiftlint:disable force_cast
+// swiftlint:disable force_unwrapping
```

Comment:
Ok I'm a bit concerned about allowing force unwrapping as this typically leads to crashes.